### DISCUSSION TITLE
A way to delete multiple events?

### DISCUSSION THREAD

**Discussion by vyruz1986** [blakeblackshear/frigate#3480] [Ask A Question] [answered]
Just wanted to check if there's something obvious I'm missing before I go deleting a bunch of events one-by-one :)

**Comment by NickM-27** [comment, marked as answer]
In the beta this is improved, but it is an open feature request

**Reply by vyruz1986** [reply]
Thanks, I assume you mean #1986? Just wanted to have it linked here in case other people come across this question

**Reply by NickM-27** [reply]
I was talking about https://github.com/blakeblackshear/frigate/issues/2063 actually
